mlflow/tracking/fluent.py

```diff
@@ -815,6 +816,26 @@ def get_experiment_by_name(name: str) -> Optional[Experiment]:
     return MlflowClient().get_experiment_by_name(name)
 
 
+def list_experiments(
+    view_type: Optional[str] = None, max_results: Optional[int] = None,
```

Comment:
Can we explicitly set `view_type` to `ACTIVE_ONLY` by default? `None` is harder for users to reason about, and most users probably only care about active experiments.

Reply:
Got it, I'll set `view_type` to `ACTIVE_ONLY`.